docs/2_1_common_api.adoc

```diff
@@ -617,16 +617,12 @@ Allowed Function Calls::
 
 ===== State: Intermediate Update Mode [[IntermediateUpdateMode]]
 
-The FMU enters this state when the FMU calls <<fmi3CallbackIntermediateUpdate>> in state *Step Mode* (CS) or in state *Clock Activation Mode* (SE).
-When the callback function returns, the FMU returns to *Step Mode* (CS) or to *Clock Activation Mode* (SE).
-
-In this state the co-simulation algorithm can retrieve information from the FMU between communication points.
 This state is only available in Co-Simulation and Scheduled Execution.
-Functions called in this state must not return <<fmi3Discard>>.
 
-The FMU enters this state by calling <<fmi3CallbackIntermediateUpdate>> within <<fmi3DoStep>> (CS) or <<fmi3ActivateModelPartition>> (SE) and leaves the state towards state *Step Mode* (CS) or *Clock Activation Mode* (SE) if the function returns <<fmi3OK>> or <<fmi3Warning>>.
+The FMU enters this state by calling <<fmi3CallbackIntermediateUpdate>> within *Step Mode* (CS) or *Clock Activation Mode* (SE) and leaves the state towards state *Step Mode* (CS) or *Clock Activation Mode* (SE) if the function returns <<fmi3OK>> or <<fmi3Warning>>.
```

Comment:
Why do you want to change this?

Reply:
> Why do you want to change this?

The second half of the sentence talks about returning to states. It seemed logical to also use the states as the starting point of the transition, not a "random" function call on the stack.